digdag-client/src/test/java/io/digdag/client/api/ModelCompatibilityTest.java

```diff
@@ -67,4 +67,96 @@ private String removeField(String json, String field)
         assertThat(node.remove(field) == null, is(false));
         return mapper.writeValueAsString(node);
     }
+
+    @Test
+    public void testSessionAttemptStatusSuccess()
+            throws Exception
+    {
+        RestSessionAttempt attempt = RestSessionAttempt.builder()
+            .id(Id.of("1"))
+            .index(3)
+            .project(IdAndName.of(Id.of("1"), "p"))
+            .workflow(NameOptionalId.of("w", Optional.of(Id.of("1"))))
+            .sessionId(Id.of("1"))
+            .sessionUuid(UUID.randomUUID())
+            .sessionTime(OffsetDateTime.now(ZoneId.of("UTC")))
+            .retryAttemptName(Optional.absent())
+            .done(false)
+            .success(true)
+            .cancelRequested(false)
+            .params(newConfig())
+            .createdAt(Instant.now())
+            .finishedAt(Optional.absent())
+            .build();
+        assertThat(attempt.getStatus(), is("success"));
+    }
+
+    @Test
+    public void testSessionAttemptStatusError()
+            throws Exception
+    {
+        RestSessionAttempt attempt = RestSessionAttempt.builder()
+            .id(Id.of("1"))
+            .index(3)
+            .project(IdAndName.of(Id.of("1"), "p"))
+            .workflow(NameOptionalId.of("w", Optional.of(Id.of("1"))))
+            .sessionId(Id.of("1"))
+            .sessionUuid(UUID.randomUUID())
+            .sessionTime(OffsetDateTime.now(ZoneId.of("UTC")))
+            .retryAttemptName(Optional.absent())
+            .done(true)
+            .success(false)
+            .cancelRequested(false)
+            .params(newConfig())
+            .createdAt(Instant.now())
+            .finishedAt(Optional.absent())
+            .build();
+        assertThat(attempt.getStatus(), is("error"));
+    }
+
+    @Test
+    public void testSessionAttemptStatusKilled()
+            throws Exception
+    {
+        RestSessionAttempt attempt = RestSessionAttempt.builder()
+            .id(Id.of("1"))
+            .index(3)
+            .project(IdAndName.of(Id.of("1"), "p"))
+            .workflow(NameOptionalId.of("w", Optional.of(Id.of("1"))))
+            .sessionId(Id.of("1"))
+            .sessionUuid(UUID.randomUUID())
+            .sessionTime(OffsetDateTime.now(ZoneId.of("UTC")))
+            .retryAttemptName(Optional.absent())
+            .done(false)
```

Comment:
In current implementation, `killed` is printed out only when the session attempt has been done...?

Reply:
Ah, found this one https://ci.appveyor.com/project/treasure-data/digdag/builds/33292140/job/ndb3n2q3x25i9wj3#L967.

Let me tweak it...